context/app/static/js/components/Search/SearchBarLayout/SearchBarLayout.jsx

```diff
@@ -1,21 +1,37 @@
+/* eslint-disable max-classes-per-file */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { SearchBox, SortingSelector, ViewSwitcherToggle } from 'searchkit';
+import { SearchBox, SelectedFilters, SortingSelector, ViewSwitcherToggle } from 'searchkit';
 
 import SearchViewSwitch from '../SearchViewSwitch';
 import TilesSortDropdown from '../TilesSortDropdown';
-import { Flex, StyledDiv } from './style';
+import { Flex, StyledDiv, StyledCancelIcon } from './style';
+
+function SelectedFilter(props) {
+  const { bemBlocks, labelKey, labelValue, removeFilter, filterId } = props;
+  return (
+    <div className={bemBlocks.option().mix(bemBlocks.container('item')).mix(`selected-filter--${filterId}`)}>
+      <div className={bemBlocks.option('name')}>
+        {labelKey}: {labelValue}
+      </div>
+      <StyledCancelIcon onClick={removeFilter} />
+    </div>
```

Comment:
Seems a bit odd to extend the seachkit styles in Search.scss and then have to add in all of the bemBlocks code. It could be simpler to just write our own styled components/css and not rely on searchkit styles, especially since we'll need to migrate to the new searchkit or find an alternative.

Reply:
good call! fixing...